jonprl-mode.el

```diff
@@ -48,6 +48,28 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   (":" . ?∶)
+   ("<" . ?⟨)
+   ("<>" . ?⋄)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   (">" . ?⟩)
+   ("[" . ?⸤)
+   ("]" . ?⸥)
```

Comment:
Interesting! If we end up figuring out such a solution, then I'm all for it! I agree that the delimiter replacements aren't that important, and I could just as soon do without them.